<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCircleDialog</class>
 <widget class="QWidget" name="CreateCircleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="CreateCircleDialogGLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="HeaderFrame" name="headerFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="frameGLayout">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="mainHLayout">
        <item>
         <widget class="QGroupBox" name="circleGBox">
          <property name="title">
           <string>Circle Membership</string>
          </property>
          <layout class="QVBoxLayout" name="circleGBoxVLayout">
           <item>
            <widget class="QTreeWidget" name="treeWidget_membership">
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>IDs</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="separatorVLayout">
          <item>
           <spacer name="separatorTopVSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>188</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="toolTip">
             <string>Add</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../images.qrc">
              <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeButton">
            <property name="toolTip">
             <string>Remove </string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../images.qrc">
              <normaloff>:/images/forward.png</normaloff>:/images/forward.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="separatorBottumVSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>178</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="knownIDGBox">
          <property name="title">
           <string>Known Identities</string>
          </property>
          <layout class="QVBoxLayout" name="knownIDGBoxVLayout">
           <item>
            <widget class="QFrame" name="frame_PgpTypes">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="frame_PgpTypesHLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="radioButton_ListAll">
                <property name="text">
                 <string>All</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_ListAllPGP">
                <property name="text">
                 <string>Signed</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_ListKnownPGP">
                <property name="text">
                 <string>Signed by known nodes</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="filterHLayout">
             <item>
              <widget class="QLabel" name="IdFilter_Label">
               <property name="text">
                <string>Filter</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="LineEditClear" name="IdFilter"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="RSTreeWidget" name="treeWidget_IdList">
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string>Nickname</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Type</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_title">
        <property name="title">
         <string>Circle Details</string>
        </property>
        <layout class="QVBoxLayout" name="groupBox_titleVLayout">
         <item>
          <layout class="QHBoxLayout" name="nameHLayout">
           <item>
            <widget class="QLabel" name="circleNameLabel">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="circleName"/>
           </item>
           <item>
            <widget class="QLabel" name="idChooserLabel">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Creator</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="GxsIdChooser" name="idChooser"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="frame_Distribution">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="frame_DistributionHLayout">
            <property name="spacing">
             <number>9</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="distributionLabel">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Distribution</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_Public">
              <property name="text">
               <string>Public</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_Self">
              <property name="text">
               <string>Self-Restricted</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_Restricted">
              <property name="text">
               <string>Restricted to:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="GxsCircleChooser" name="circleComboBox"/>
            </item>
            <item>
             <spacer name="distributionHSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>147</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditClear</class>
   <extends>QLineEdit</extends>
   <header location="global">gui/common/LineEditClear.h</header>
  </customwidget>
  <customwidget>
   <class>HeaderFrame</class>
   <extends>QFrame</extends>
   <header>gui/common/HeaderFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GxsIdChooser</class>
   <extends>QComboBox</extends>
   <header>gui/gxs/GxsIdChooser.h</header>
  </customwidget>
  <customwidget>
   <class>RSTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>gui/common/RSTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>GxsCircleChooser</class>
   <extends>QComboBox</extends>
   <header>gui/gxs/GxsCircleChooser.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
